crypto/keys/lazy_keybase_test.go

```diff
@@ -57,7 +57,7 @@ package keys
 // 	require.NotNil(t, err)
 // 	_, err = kb.GetByAddress(accAddr(i2))
 // 	require.NoError(t, err)
-// 	addr, err := sdk.AccAddressFromBech32("cosmos1yq8lgssgxlx9smjhes6ryjasmqmd3ts2559g0t")
+// 	addr, err := sdk.AddressFromBech32("cosmos1yq8lgssgxlx9smjhes6ryjasmqmd3ts2559g0t")
 // 	require.NoError(t, err)
 // 	_, err = kb.GetByAddress(addr)
 // 	require.NotNil(t, err)
```

Comment:
@andrewnguyen22 can we just remove these tests ?

Reply:
Actually, it would be nice if we implemented them.
https://github.com/pokt-network/posmint/issues/66